docs/reference/issuers/acme/dns01/infoblox.rst

```diff
@@ -0,0 +1,22 @@
+=========================
+Infoblox
+=========================
+
+This provider uses the fields mentioned in the following example in
+order to work. Also it uses a Kubernetes ``Secret`` Resource. In the
+following example, the secret will have to be named ``infoblox``
+and have a subkey ``password`` with the password in it.
+
+
+.. code-block:: yaml
+
+   infoblox:
+     gridHost: "infoblox.example.com"
+     wapiUsername: "infoblox_user"
+     wapiPort: 443
```

Comment:
Yeap it would be easier for the user. I'm not sure if the `infoblox-go-client` supports `https://` or `http://` in front of the url though. I'll have to check. I will need also to implement a checker to check if the provided `gridHost` is in the right form I guess.

Reply:
If you take a look in the `validation` package, you should be able to see a spot to add that check! 😄